app/controllers/employees_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
   end
 
   def create
-    @employee = Employee.create(employee_params)
+    @employee = Employee.new(employee_params)
```

Comment:
No need for creating a class variable if you use it only inside the method. `@employee` -> `employee`